src/Shell/Task/SeedTask.php

```diff
@@ -140,8 +164,90 @@ public function getOptionParser()
             'choices' => $bakeThemes
         ])->addArgument('name', [
             'help' => 'Name of the seed to bake. Can use Plugin.name to bake plugin models.'
+        ])->addOption('data', [
+            'short' => 'd',
+            'boolean' => true,
+            'help' => 'Include data from the table to the seed'
+        ])->addOption('fields', [
+            'short' => 'f',
+            'default' => '*',
+            'help' => 'If including data, comma separated list of fields to select (all fields by default)',
+        ])->addOption('limit', [
+            'short' => 'l',
+            'help' => 'If including data, max number of rows to select'
         ]);
 
         return $parser;
     }
+
+    /**
+     * Prettify var_export of an array output
+     *
+     * @param array $array              Array to prettify
+     * @param int $tabCount             Initial tab count
+     * @param string $indentCharacter   Desired indent for the code.
+     * @return string
+     */
+    protected function prettifyArray($array, $tabCount = 3, $indentCharacter = "    ")
+    {
+        $content = var_export($array, true);
+
+        $lines = explode("\n", $content);
+
+        $inString = false;
+
+        foreach ($lines as $k => &$line) {
+            if ($k == 0) {
```

Comment:
those two can also be strict, as the key must be numeric (coming from explode function).